src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,6 +309,16 @@ export class RunParametersAction extends TooltipFromLabelAction {
 	*/
 	public async run(context: URI): Promise<void> {
 		const editor = this._notebookService.findNotebookEditor(context);
+		// Only run action for kernels that are supported (Python, PySpark, PowerShell)
+		let kernelsNotSupported = ['sql', 'kusto'];
```

Comment:
It would be better to use an allow list here - because right now if we added another kernel (that's unsupported, as most would be I imagine) then we'd have to remember to go in and update this.